### PR TITLE
⚙️Integrate dotenv

### PR DESCRIPTION
**Pull request by alihaiderrizvi**
Integrated dotenv with the codebase.

### CODE REVIEW

File: apis/artsy.py [hunk 1]
```diff
@@ -1,8 +1,11 @@
 import requests
 from typing import Dict
-from constants import artsy_client_id, artsy_client_skey, artsy_api_url
 import logging
+import os
 
+artsy_client_id = os.getenv("ARTSY_CLIENT_ID")
+artsy_client_skey = os.getenv("ARTSY_CLIENT_SKEY")
+artsy_api_url = os.getenv("https://api.artsy.net/api")
```
Comment (hxghhhh) [inline apis/artsy.py]: shouldn't this just be the url?
```suggestion
artsy_api_url = "https://api.artsy.net/api"
```

**Comment by hxghhhh** [comment]
we should only make the env hold the keys urls we should bake on our side in a constant.py file or just keep it in the code

File: utils/enrichment.py [hunk 2]
```diff
@@ -8,6 +8,8 @@
 from apis.wikipedia import fetch_wikipedia
 from typing import Dict
 from constants import APIs
+import os
```
Comment (joelrojo) [inline utils/enrichment.py]: @alihaiderrizvi not sure we need this import here?

**Review by joelrojo — CHANGES_REQUESTED** [review]
@alihaiderrizvi looking good! if you can just get rid of that import (unless we need it). Also can you add the environment keys to this notion doc:
https://www.notion.so/Mercury-API-keys-2bcae4d6433d477d8c5cb89f732afcfe

Similar to the way I did in this notion doc: https://www.notion.so/Mainframe-API-Keys-9c56f75680d64f35ad6b42f8cb1d6a8b

**Review by joelrojo — APPROVED** [review]

**Pull request merged**